contracts/interfaces/modules/royalty/policies/IIpPool.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+import { IERC20 } from "@openzeppelin/contracts-v4/token/ERC20/IERC20.sol";
+
+/// @title Ip pool interface
+interface IIpPool is IERC20 {
```

Comment:
IPPool is a kinda confusing name, can we name it IPRoyaltyVault?

Reply:
Addressed here: https://github.com/storyprotocol/protocol-core-v1/pull/26/commits/bf10d6ac3dafcc8864f5aedc2edf2629fffe2be0